examples/rack.rb

```diff
@@ -0,0 +1,46 @@
+# (c) Copyright 2016 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+client = {
+  url: '',
+  user: '',
+  password: ''
+}
+
+oneview_rack 'Rack_1' do
```

Comment:
I like all the examples, but I think we need some comments in here explaining what's going on. A few things that I think need commenting on:

- In this example, the name attribute is the rack name we'll search for on OneView. That might not be clear to new users.
- In the add_to and remove_from rack actions, users will need to know that the name and type attributes are special (and are there certain valid/invalid types?). Also, any additional hash key-pairs are passed to the `add_rack_resource` method

The idea is that a new user can read through the example and be able to understand how and why the resource works the way it does.

I also try to write them as working recipes (minus the fake credentials), meaning no duplicating resource names, etc that might cause the Chef resource compiler to fail. If you comment about it, it'll be clear to users that they can't add a rack and add a resource to the rack, using the same resource name for each.